plugins/org-react/src/components/GroupListPicker/GroupListPickerButton.tsx

```diff
@@ -27,13 +27,18 @@ const useStyles = makeStyles((theme: BackstageTheme) => ({
     width: '100%',
     cursor: 'pointer',
     justifyContent: 'space-between',
+    '& p': {
```

Comment:
Could you explain what this does, I'm not sure what `& p` represent in the context of the picker 😅

Reply:
as this styles the button, the `& p` corresponds to the text part of the button.

Reply:
Alright. Since this is a MUI component and its internal implementation might in theory change over time, we should do that kind of styling in their preferred way instead.

https://v4.mui.com/api/button/

So, render the button as `<Button classes={{ label: classes.buttonLabel }}` and make a `buttonLabel` style here

Reply:
Good suggestion, but it does not really fix the problem without pointing to the child paragraf element as the label in the classes object corresponds with the parent span element that wraps the icon (in a span), the text (in a paragraf) and the chevron icon (in a span). See screenshot below:

<img width="593" alt="Screenshot 2022-11-21 at 10 05 24" src="https://user-images.githubusercontent.com/7124145/203009578-9bfe57d3-f778-47bf-82e5-44ab041db931.png">
<img width="586" alt="Screenshot 2022-11-21 at 09 59 20" src="https://user-images.githubusercontent.com/7124145/203009583-cd49e3ee-8860-433f-a0fd-502d4ff988c6.png">

Do you want me to make change suggested in the screenshot or not @freben ?

Reply:
Alright, then style the `Typography` which generates the `p` instead, right?

Reply:
Yes, i could just style the `Typography`, don't know why i missed that =). 
Fixed